code/pom.xml

```diff
@@ -33,6 +33,16 @@
         <artifactId>appengine-api-1.0-sdk</artifactId>
         <version>1.9.59</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.apis</groupId>
+      <artifactId>google-api-services-gmail</artifactId>
+      <version>v1-rev20200601-1.30.10</version>
```

Comment:
Can you try to sort these alphabetically, so it's easier to check for duplicates/inclusion?